dashboards/grafana-dashboard-assisted-resource-dashboard.yaml

```diff
@@ -698,7 +733,7 @@ data:
       "timezone": "",
       "title": "Assisted Service Resurce Dashboard",
```

Comment:
Update title and  uid
 "title": "Assisted Service Resource Dashboard",
  "uid": "assisted-service-resource-dashboard",

Reply:
updates applied.